### ISSUE TITLE
constexpr in SafeInt3.hpp kills the compiling on vc2013

### ISSUE THREAD

**Issue by bbqz007** [microsoft/cpprestsdk#1608]
Release\include\cpprest\details\SafeInt3.hpp
```constexpr SafeInt(const SafeInt<T, E>& u) SAFEINT_CPP_THROW = default;```

and i find ```CPPREST_CONSTEXPR``` by which cpprest solves the problem that vc120 does not fully support constexpr.

**Comment by abcdefghi123456jk** [comment]
you have to upgrade your VS to 2015 update 3 or later though

**Comment by bbqz007** [comment]
yes, i know the vs2015 support constexpr. and cpprestsdk project uses CPPREST_CONSTEXPR to solves the problems cause of constexpr. i think replace that with CPPREST_CONSTEXPR to make more friendly  to the lower versions of  vs, just like this project did before.

**Comment by jlaurila** [comment]
I would like to compile against vc120 too. Commenting out this one line seems to work so would it be possible to add a directive for conditional compilation of this?
